### PR TITLE
Add immedia ui hooks

### PR DESCRIPTION
**Pull request by luisbosch**
Added usernames under each video feed.

### CODE REVIEW

File: app/js/immedia/immedia-room-controller.js [hunk 1]
```diff
@@ -6,7 +6,18 @@
...
+    '$scope',
+    '$sce',
+    '$window',
+    '$routeParams',
+    '$interval',
+    'RoomService',
+    'ConfigService',
+    'AppUsersManager',
+
+    function($rootScope, $scope, $sce, $window, $routeParams, $interval, roomSvc, cfgSvc, AppUsersManager) {
```
Comment (adamantivm) [inline app/js/immedia/immedia-room-controller.js]: Please rename `AppUsersManager` to `appUsersManager` for consistency

File: app/js/immedia/immedia-room-controller.js [hunk 2]
```diff
@@ -6,7 +6,18 @@
 
   var immediaControllers = angular.module('immediaControllers');
 
-  immediaControllers.controller('RoomCtrl', ['$rootScope', '$scope', '$sce', '$window', '$routeParams', '$interval', 'RoomService', 'ConfigService', function($rootScope, $scope, $sce, $window, $routeParams, $interval, roomSvc, cfgSvc) {
+  immediaControllers.controller('RoomCtrl', ['$rootScope',
```
Comment (adamantivm) [inline app/js/immedia/immedia-room-controller.js]: I'd prefer having this list in a single line, with a break before the function definition maybe.

**Comment by adamantivm** [comment]
Thanks for the contribution @luisbosch ! I made a few minor comments. If you could fix those and let me know I'll merge and we can deploy.

**Comment by luisbosch** [comment]
Fixed findings!

**Comment by adamantivm** [comment]
:+1:

**Pull request merged**